sections/swym-custom-wishlist.liquid

```diff
@@ -0,0 +1,1843 @@
+                                  selectedVariant
+                                    ? selectedVariant.available
+                                      ? isInCart
+                                        ? `{{ settings.swym_wishlist_item_cta_addedtocart }}`
+                                        : `{{ settings.swym_wishlist_item_cta_addtocart }}`
+                                      : `{{ settings.swym_wishlist_item_cta_soldout }}`
+                                    : `{{ settings.swym_wishlist_item_cta_unavailable }}`
+                                }</button>`
+                              : ''
+                          }
+                      </div>
+                  </div>
+                  ${
+                    settings.swym_wishlist_show_delete_item
+                      ? `
+                    <button id="swymcs-remove-productBtn" aria-label="Delete" class="swymcs-wishlist-item-remove">
+                      <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
+                        <path d="M5.43742 17.5C5.10409 17.5 4.81242 17.375 4.56242 17.125C4.31242 16.875 4.18742 16.5833 4.18742 16.25V4.375H3.95825C3.78117 4.375 3.63273 4.31476 3.51294 4.19427C3.39315 4.07377 3.33325 3.92447 3.33325 3.74635C3.33325 3.56823 3.39315 3.42014 3.51294 3.30208C3.63273 3.18403 3.78117 3.125 3.95825 3.125H7.24992C7.24992 2.94444 7.30981 2.79514 7.42961 2.67708C7.5494 2.55903 7.69783 2.5 7.87492 2.5H12.1249C12.302 2.5 12.4504 2.5599 12.5702 2.67969C12.69 2.79948 12.7499 2.94792 12.7499 3.125H16.0416C16.2187 3.125 16.3671 3.18524 16.4869 3.30573C16.6067 3.42623 16.6666 3.57553 16.6666 3.75365C16.6666 3.93177 16.6067 4.07986 16.4869 4.19792C16.3671 4.31597 16.2187 4.375 16.0416 4.375H15.8124V16.25C15.8124 16.5833 15.6874 16.875 15.4374 17.125C15.1874 17.375 14.8958 17.5 14.5624 17.5H5.43742ZM5.43742 4.375V16.25H14.5624V4.375H5.43742ZM7.64575 13.8333C7.64575 14.0104 7.70599 14.1589 7.82648 14.2786C7.94698 14.3984 8.09629 14.4583 8.2744 14.4583C8.45252 14.4583 8.60061 14.3984 8.71867 14.2786C8.83672 14.1589 8.89575 14.0104 8.89575 13.8333V6.77083C8.89575 6.59375 8.83551 6.44531 8.71502 6.32552C8.59452 6.20573 8.44522 6.14583 8.26711 6.14583C8.08898 6.14583 7.94089 6.20573 7.82283 6.32552C7.70478 6.44531 7.64575 6.59375 7.64575 6.77083V13.8333ZM11.1041 13.8333C11.1041 14.0104 11.1643 14.1589 11.2848 14.2786C11.4053 14.3984 11.5546 14.4583 11.7327 14.4583C11.9109 14.4583 12.0589 14.3984 12.177 14.2786C12.2951 14.1589 12.3541 14.0104 12.3541 13.8333V6.77083C12.3541 6.59375 12.2938 6.44531 12.1734 6.32552C12.0529 6.20573 11.9036 6.14583 11.7254 6.14583C11.5473 6.14583 11.3992 6.20573 11.2812 6.32552C11.1631 6.44531 11.1041 6.59375 11.1041 6.77083V13.8333Z" fill="#5C5F62"/>
+                      </svg>
+                    </button>
+                    `
+                      : ''
+                  }
+                  `;
+
+        this.attachListner();
```

Comment:
this will re-render entier html again, so again event will attach to button, always only one event will be attached to button, and it's essential, i don't think it will lead to memory leaks.